src/main/kotlin/com/beust/klaxon/DefaultConverter.kt

```diff
@@ -44,6 +44,10 @@ class DefaultConverter(private val klaxon: Klaxon, private val allPaths: HashMap
         val result = when (value) {
             is String, is Enum<*> -> "\"" + Render.escapeString(value.toString()) + "\""
             is Double, is Float, is Int, is Boolean, is Long -> value.toString()
+            is Array<*> -> {
+                val elements = value.filterNotNull().map { klaxon.toJsonString(it) }
```

Comment:
If I understand this correctly, when serializing an array, null elements aren't printed?
Why is this done?
JSON arrays can have null objects.
Serializing "listOf(1, 2, null, null, 3)" should produce "[1, 2, null, null, 3]", right?

Reply:
... Same goes for Collection down below